.github/workflows/linting.yml

```
@@ -28,10 +28,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repository
-        uses: actions/checkout@v2
+        Uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
```

`Uses` breaks the job definition.

```suggestion
        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
```

---

Yikes, good catch. I didn't seem to do this typo elsewhere, grepping around.